### DISCUSSION TITLE
How do you remove the "App in production" message ?

### DISCUSSION THREAD

**Discussion by prosticoco** [dokku/dokku#4655] [Q&A]
Hello, I am trying to write a script that automates dokku installation and initial deployment of a Laravel app on a remote server. The issue is, I need to run commands to migrate the tables and seed the database after the app is deployed and want to automate running the `php artisan` commands in a bash script: but I cannot automate as I do not know how to avoid the "App in Production" warning message, how do I disable that ?

I have already tried running `docker ps:stop <app-name>` before running the commands and it still does not work. I would also want to avoid using dirty tools such as `expect` if possible.

Thanks !

**Comment by josegonzalez** [comment]
What is the "App in Production" warning? Do you have a full example of what you're seeing?
